conda/recipes/dask-cuda/meta.yaml

```
@@ -41,6 +41,13 @@ requirements:
 test:
   imports:
     - dask_cuda
+  commands:
+    {% for e in data.get("project", {}).get("scripts", {}).keys() %}
+    - {{ e }} --help
+    {% endfor %}
+    - dask cuda --help
+    - dask cuda worker --help
+    - dask cuda config --help
```

Actually we might be able to do this to consolidate them further

```suggestion
    - dask cuda --help
    {% for e in data.get("project", {}).get("scripts", {}).keys() %}
    - {{ e }} --help
    - {{ e|replace("-", " ") }} --help
    {% endfor %}
```

---

Good catch! Yeah for now we can do this, interested in if we intend to deprecate and eventually remove the old entrypoints, which would force us to hardcode this some time in the future (cc @jacobtomlinson if you've had any discussion around this)

---

Yeah that's fair. For that reason wasn't sure whether we would want to do this or not